public/pages/DetectorResults/containers/AnomalyResultsTable.tsx

```diff
@@ -80,7 +104,175 @@ export function AnomalyResultsTable(props: AnomalyResultsTableProps) {
       ? props.anomalies.filter((anomaly) => anomaly.anomalyGrade > 0)
       : [];
 
+  // find feature output values for a specific anomaly
+  const getFeatureOutputValues = (anomalyTime: number, anomalyEntity?: any) => {
+    const featureValues: { [featureName: string]: number } = {};
+    
+    if (!props.anomalyAndFeatureResults || !props.detector?.featureAttributes) {
+      return featureValues;
+    }
+
+    // find the feature data that matches the anomaly time and entity
+    props.anomalyAndFeatureResults.forEach((timeSeries: any) => {
+      if (timeSeries.featureData) {
+        Object.keys(timeSeries.featureData).forEach((featureId: string) => {
+          const featureDataPoints = timeSeries.featureData[featureId];
+          
+          const matchingFeaturePoint = featureDataPoints.find((point: any) => {
+            return Math.abs(point.plotTime - anomalyTime) < 30000; // 30 second tolerance
+          });
+          
+          if (matchingFeaturePoint) {
+            const featureConfig = props.detector.featureAttributes.find((attr: any) => 
+              attr.featureId === featureId
+            );
+            
+            if (featureConfig) {
+              let fieldName = featureConfig.featureName || featureId;
+              let aggregationMethod = '';
+              
+              if (featureConfig.aggregationQuery) {
+                try {
+                  const aggregationKeys = Object.keys(featureConfig.aggregationQuery);
+                  if (aggregationKeys.length > 0) {
+                    const firstAggregation = featureConfig.aggregationQuery[aggregationKeys[0]];
+                    const methodKeys = Object.keys(firstAggregation);
+                    if (methodKeys.length > 0) {
+                      aggregationMethod = methodKeys[0];
+                      const method = firstAggregation[methodKeys[0]];
+                      if (method && method.field) {
+                        fieldName = method.field;
+                      }
+                    }
+                  }
+                } catch (error) {
+                  console.warn('Error extracting field name from aggregation query:', error);
+                }
+              }
+              
+              // Only add feature filter for min/max aggregation methods
+              if (aggregationMethod === 'min' || aggregationMethod === 'max') {
+                featureValues[fieldName] = matchingFeaturePoint.data;
+              }
+            } else {
+              // fallback: use the feature name from the data point if available
+              const fieldName = matchingFeaturePoint.name || featureId;
+              featureValues[fieldName] = matchingFeaturePoint.data;
+            }
+          }
+        });
+      }
```

Comment:
If there is no feature definition, should we just leave featureValues empty and not use feature value as filter in the Discover page?